packages/valory/skills/termination_abci/behaviours.py

```diff
@@ -530,8 +529,6 @@ def is_round_ended(self, round_id: str) -> Callable[[], bool]:
 class TerminationBehaviour(BaseBehaviour):
     """Behaviour responsible for terminating the agent."""
 
-    state_id: str = "termination"
```

Comment:
Another mysterious `state_id`

Reply:
The scaffold has it by default I believe, here we use it all over the place https://github.com/valory-xyz/autonomous-fund/blob/main/packages/balancer/skills/fear_and_greed_oracle_abci/behaviours.py